### ISSUE TITLE
The system cannot find the file specified

### ISSUE THREAD

**Issue by Hanak85** [gophish/gophish#966]
**Thanks for reaching out! We're happy to help resolve issues as quickly as possible.**

**Please use this template when creating a new issue. If you do not follow this template format, your issue may be closed without comment.**

Before filing a new issue, please use the search bar at the top of the browser to search for similar issues. Also, please make sure you have read our [documentation](https://getgophish.com/documentation/), which covers how to use Gophish.

What version of Gophish are you using?: 0.5.0

Brief description of the issue: Running gophish.exe closes immediately. ran in CMD and i get the message gophish.go:59: open ./VERSION: The system cannot find the file specified. was working previously. tried in powershell in reference to #889  with same results. 

What are you expecting to see happen? : gophish run

What are you seeing happen? message with cant find version file. its in the root folder with .exe

If this question is related to email templates or landing pages not working as expected, please provide your template or landing page below:

```html
Insert code here
```C:\>"Program Files\gophish-v0.5.0-windows-64bit\gophish.exe"

Please provide any terminal output that may be relevant below:
 2018/02/08 16:16:32 worker.go:26: Background Worker Started Successfully - Waiting for Campaigns
 2018/02/08 16:16:32 gophish.go:59: open ./VERSION: The system cannot find the file specified.
```
Insert output here
```

Please provide as many steps as you can to reproduce the problem:
 * Step 1 - 
 * Step 2 -

**Comment by jordan-wright** [comment]
Can you print out the contents of the directory? As in, what files are in there?

It seems that the `VERSION` file isn't in that directory, yet I've just confirmed that it's present in the release files. If you do see the file listed, you might also check file permissions, since I see gophish was placed in your Program Files.

**Comment by Hanak85** [comment]
Directory of C:\Program Files\gophish-v0.5.0-windows-64bit

02/08/2018  04:08 PM    <DIR>          .
02/08/2018  04:08 PM    <DIR>          ..
01/27/2018  03:38 AM               373 config.json
02/08/2018  04:06 PM    <DIR>          db
02/08/2018  04:08 PM            73,728 gophish.db
01/27/2018  03:44 AM        16,063,451 gophish.exe
01/27/2018  03:38 AM             1,115 LICENSE
01/27/2018  03:38 AM             3,078 README.md
02/08/2018  04:06 PM    <DIR>          static
02/08/2018  04:06 PM    <DIR>          templates
01/27/2018  03:38 AM                 5 VERSION
               6 File(s)     16,141,750 bytes
               5 Dir(s)  485,505,089,536 bytes free

**Comment by jordan-wright** [comment]
Hmm... the file is definitely there. I'm wondering if perhaps it's a file permissions issue. What happens if you open cmd (you might also try Powershell - I've seen reports of mixed results before) and manually navigate to the gophish directory and try to execute the binary?

**Comment by Hanak85** [comment]
Both cmd and powershell produce same results. even ran as admin

**Comment by Hanak85** [comment]
tried on secondary admin profile with same results.

**Comment by Hanak85** [comment]
Enabled IIS to create cert and it takes over port 80, changing the port in the config worked only after running it from a new local admin account(gave new error message about port 80) disabled IIS and ran from original location and set back to port 80 and ran successfully. thanks for you help! :)

**Comment by jordan-wright** [comment]
Glad you were able to get it resolved!
